File: SDKs/onsched.com/setup/v1/python/src/sdk/models/shared/timezonesviewmodel.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class TimezonesViewModel:
    name: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('name') }})
    region: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('region') }})
    timezone_ianna: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('timezoneIanna') }})
    tz_offset: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('tzOffset') }})
    

File: SDKs/onsched.com/setup/v1/typescript/src/sdk/models/shared/timezonesviewmodel.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class TimezonesViewModel extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=name" })
  name?: string;

  @SpeakeasyMetadata({ data: "json, name=region" })
  region?: string;

  @SpeakeasyMetadata({ data: "json, name=timezoneIanna" })
  timezoneIanna?: string;

  @SpeakeasyMetadata({ data: "json, name=tzOffset" })
  tzOffset?: number;
}
